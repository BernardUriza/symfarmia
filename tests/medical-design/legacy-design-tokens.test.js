const path = require('path');

describe('Legacy design tokens', () => {
  const tokens = require(path.resolve(
    __dirname,
    '../../legacy-design/design-tokens/medical-theme.json'
  ));

  it('should match the approved medical design tokens', () => {
    expect(tokens).toMatchSnapshot();
  });
});