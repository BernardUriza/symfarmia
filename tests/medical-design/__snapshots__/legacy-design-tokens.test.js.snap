// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Legacy design tokens should match the approved medical design tokens 1`] = `{}`;
